docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>Please put the first six lines in a seperate cell.</p><p> </p><blockquote>type(flsoat)</blockquote><p> </p><p>Does this work? I think there is a typo here, and it should have been <code>float</code> . right? I think you should run the cells one last time before pushing them to this PR to make sure everthing is working properly.</p><p> </p><blockquote>ecg_demo_df</blockquote><p> </p><p>Are we using this later? If not, then a simple name like <code>df</code> should suffice.</p><pre class="ql-syntax" spellcheck="false">df = pd.read_csv(...).astype(...)
T_m = df.values[:,0]
T_p = df.values[:,1] 
</pre><p> </p><p>What is <code>m</code> in <code>T_m </code> ? Did you mean <code>T_n</code> (because of the word <code>negative</code>) If yes, then please use <code>T_neg</code> and <code>T_pos</code> . Having said that, I think we should avoid positive / negative thing and just use <code>T </code> since we haven't discussed pos/neg yet. So, based on what I can see in the next few cells, it seems that we are not using <code>T_m </code>. So, maybe we should just do:</p><pre class="ql-syntax" spellcheck="false">T = df.values[:,1]  
</pre><p> </p><p>Then, discuss the problem and the challenge on this data <code>T</code>. </p><p> </p><blockquote>MP_PP1 = stumpy.stump(T_p[:2000], m, normalize=False)</blockquote><p> </p><p>Can we just use the name <code>mp</code> ?    </p><p> </p><blockquote>discord_idx1 = np.argmax(MP_PP1[:, 0])&nbsp;# disccords</blockquote><p>Can we just say: <code>discord_idx </code> ? Also, there is a typo in your comment: <code>disccords</code> has one <code>c</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>